usbhost.h

```diff
@@ -553,6 +573,9 @@ uint8_t MAX3421e< SPI_SS, INTR >::Task(void) {
         //    }
         //    usbSM();                                //USB state machine
         return ( rcode);
+        */
+        busprobe();
```

Comment:
Maybe add a comment why the code above was commented out and replaced with just `busprobe`.

Reply:
Is this proper for the comment? If so I'll commit it.

```
// CONDETIRQ can miss detection of plugin and check bus every time anyway #653
```

Reply:
What about something like this:

```C++
/* MAX3421 state change task and interrupt handler */
template< typename SPI_SS, typename INTR >
uint8_t MAX3421e< SPI_SS, INTR >::Task(void) {
#if 0
        uint8_t rcode = 0;
        uint8_t pinvalue;
        //USB_HOST_SERIAL.print("Vbus state: ");
        //USB_HOST_SERIAL.println( vbusState, HEX );
        pinvalue = INTR::IsSet(); //Read();
        //pinvalue = digitalRead( MAX_INT );
        if(pinvalue == 0) {
                rcode = IntHandler();
        }
        //    pinvalue = digitalRead( MAX_GPX );
        //    if( pinvalue == LOW ) {
        //        GpxHandler();
        //    }
        //    usbSM();                                //USB state machine
        return ( rcode);
#else
        // Seems like MAX3421e CONDETIRQ (connect/disconnect Interrupt) doesn't arise for every plugin event, 
        // so instead we check the bus state at every call to Task when no device is connected
        // See: https://github.com/felis/USB_Host_Shield_2.0/pull/653
        busprobe();
#endif
        return 0;
}
```

I think the above code makes it more clear what the old vs new code is as well.

Reply:
It looks good!
Please feel free to add it as new commit or change existent commit before merging.

Thanks for your service on this library as always!